[OYPD-425] Changes to alert slider to make navigation easier

david-hernandez opened the pull request:
- [x] Check the navigation elements on the alert slider.
- [x] Check usability on mobile touch device.

The navigation elements were hidden on smaller sizes. I added them back. I made the arrows larger, and spaced farther apart so they are easier to tap on a touch screen. I gave the arrows and close button a more defined border instead of shadow so that it is easier to see and stands out better on different color backgrounds. I removed the transparency. I also added hover color change to the arrows so they can be seen as clickable. And I updated the color of the close button hover to match the arrow hover color.

![screen shot 2017-06-02 at 4 39 49 pm](https://cloud.githubusercontent.com/assets/1504038/26744217/ff6784ae-47b2-11e7-8c1c-2b9540961734.png)
![screen shot 2017-06-02 at 4 39 44 pm](https://cloud.githubusercontent.com/assets/1504038/26744216/ff670d80-47b2-11e7-96bc-7e03747d3e58.png)
![screen shot 2017-06-02 at 4 39 37 pm](https://cloud.githubusercontent.com/assets/1504038/26744215/ff671ca8-47b2-11e7-9894-88d515b309f6.png)
![screen shot 2017-06-02 at 4 39 28 pm](https://cloud.githubusercontent.com/assets/1504038/26744219/ff6ee514-47b2-11e7-8d92-14d1d2ae0b73.png)
![screen shot 2017-06-02 at 4 39 20 pm](https://cloud.githubusercontent.com/assets/1504038/26744218/ff6b2438-47b2-11e7-8830-7756fa040c50.png)
![screen shot 2017-06-02 at 4 46 45 pm](https://cloud.githubusercontent.com/assets/1504038/26744251/17ec4c62-47b3-11e7-89d6-e211456a4e7a.png)

david-hernandez committed:
Changes to alert slider to make navigation easier

gundevel commented:
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1343 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1343 |
| Installation process | http://install.openy-dev.ffwua.com/build1343/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1343/

gundevel commented:
Behat test results file URL: http://ci.openymca.org/build1343/profiles/contrib/openy/build/reports/behat

Sanchiz commented:
retest this please

gundevel commented:
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1344 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1344 |
| Installation process | http://install.openy-dev.ffwua.com/build1344/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1344/

gundevel commented:
Behat test results file URL: http://ci.openymca.org/build1344/profiles/contrib/openy/build/reports/behat

AlexNetman commented:
Add text "Close Alert" for the cross-icon.
Focus should jump to the alert body after Next arrow is clicked.
Remove broken aria reference: 
Example: aria-describedby="slick-slide02" but there is not slick-slide02 description actually.

david-hernandez committed:
Merge branch '8.x-1.x' into OYPD-425

david-hernandez committed:
Adding visually hidden close alert text to alert templates

david-hernandez committed:
Updating alert JS to remove aria-describedby

david-hernandez commented:
Added "Close alert" text and removed the broken aria describedby. This had to be done in JS because it is actually being added by the slick library JS directly.

david-hernandez commented:
I'll set to needs review for at least those parts.

gundevel commented:
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1512 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1512 |
| Installation process | http://install.openy-dev.ffwua.com/build1512/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1512/

gundevel commented:
Behat test results file URL: http://ci.openymca.org/build1512/profiles/contrib/openy/build/reports/behat

david-hernandez committed:
Changing buttons to links for the alert slider

david-hernandez commented:
Added an alter to get a version of the slick slider template just for alert node type sliders. I tried a couple different options but the most straightforward way to change the buttons to links was with a string replace, because the buttons are hard-coded in the JS settings prebuilt with all the attributes. :(
There is a slight CSS change to adjust the link position.

gundevel commented:
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1513 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1513 |
| Installation process | http://install.openy-dev.ffwua.com/build1513/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1513/

gundevel commented:
Behat test results file URL: http://ci.openymca.org/build1513/profiles/contrib/openy/build/reports/behat

alexschedrov commented on themes/openy_themes/openy_rose/templates/node/node--alert--node-alert-footer.html.twig:
```diff
@@ -103,7 +103,7 @@
         </div>
       {% endif %}
-      <a href="#" class="site-alert__dismiss"><i class="fa fa-times" aria-hidden="true"></i></a>
+      <a href="#" class="site-alert__dismiss"><i class="fa fa-times"><span class="visually-hidden">Close alert</span></i></a>
```
`Close alert` should be translatable.

alexschedrov commented on themes/openy_themes/openy_rose/templates/node/node--alert--node-alert-header.html.twig:
```diff
@@ -103,7 +103,7 @@
         </div>
       {% endif %}
-      <a href="#" class="site-alert__dismiss"><i class="fa fa-times" aria-hidden="true"></i></a>
+      <a href="#" class="site-alert__dismiss"><i class="fa fa-times"><span class="visually-hidden">Close alert</span></i></a>
```
`Close alert` should be translatable.

david-hernandez committed:
Adding translation to the close alert text

david-hernandez committed:
Need to specify height to override the slick css in some cases

gundevel commented:
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1560/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #434 of commit d564906e7f0d2de1490fac5079f6e0719854a90e automatically merged.
GitHub pull request #434 of commit d564906e7f0d2de1490fac5079f6e0719854a90e automatically merged.
GitHub pull request #434 of commit d564906e7f0d2de1490fac5079f6e0719854a90e automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git2816132236529860973.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
No valid HEAD. Skipping the resetting
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git6346397988073322038.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git rev-parse origin/pr/434/merge^{commit} # timeout=10
 > git branch -a --contains 8743694b67a2d9c6b5faef507c57bcf7fc29356f # timeout=10
 > git rev-parse remotes/origin/pr/434/merge^{commit} # timeout=10
Checking out Revision 8743694b67a2d9c6b5faef507c57bcf7fc29356f (origin/pr/434/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 8743694b67a2d9c6b5faef507c57bcf7fc29356f
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson3074265170305560688.sh
+ rm -Rf /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm
+ cp -R ../vm /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm
+ cd /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm
+ git pull
From github.com:propeoplemd/openy_vm
   ce3ae8d..b0a8a31  master     -> origin/master
Updating ce3ae8d..b0a8a31
Fast-forward
 .../libraries/colordialog/dialogs/colordialog.css  |   20 +
 .../libraries/colordialog/dialogs/colordialog.js   |  343 +++
 docroot/libraries/colordialog/lang/af.js           |   11 +
 docroot/libraries/colordialog/lang/ar.js           |   11 +
 docroot/libraries/colordialog/lang/az.js           |   11 +
 docroot/libraries/colordialog/lang/bg.js           |   11 +
 docroot/libraries/colordialog/lang/bn.js           |   11 +
 docroot/libraries/colordialog/lang/bs.js           |   11 +
 docroot/libraries/colordialog/lang/ca.js           |   11 +
 docroot/libraries/colordialog/lang/cs.js           |   11 +
 docroot/libraries/colordialog/lang/cy.js           |   11 +
 docroot/libraries/colordialog/lang/da.js           |   11 +
 docroot/libraries/colordialog/lang/de-ch.js        |   11 +
 docroot/libraries/colordialog/lang/de.js           |   11 +
 docroot/libraries/colordialog/lang/el.js           |   11 +
 docroot/libraries/colordialog/lang/en-au.js        |   11 +
 docroot/libraries/colordialog/lang/en-ca.js        |   11 +
 docroot/libraries/colordialog/lang/en-gb.js        |   11 +
 docroot/libraries/colordialog/lang/en.js           |   11 +
 docroot/libraries/colordialog/lang/eo.js           |   11 +
 docroot/libraries/colordialog/lang/es.js           |   11 +
 docroot/libraries/colordialog/lang/et.js           |   11 +
 docroot/libraries/colordialog/lang/eu.js           |   11 +
 docroot/libraries/colordialog/lang/fa.js           |   11 +
 docroot/libraries/colordialog/lang/fi.js           |   11 +
 docroot/libraries/colordialog/lang/fo.js           |   11 +
 docroot/libraries/colordialog/lang/fr-ca.js        |   11 +
 docroot/libraries/colordialog/lang/fr.js           |   11 +
 docroot/libraries/colordialog/lang/gl.js           |   11 +
 docroot/libraries/colordialog/lang/gu.js           |   11 +
 docroot/libraries/colordialog/lang/he.js           |   11 +
 docroot/libraries/colordialog/lang/hi.js           |   11 +
 docroot/libraries/colordialog/lang/hr.js           |   11 +
 docroot/libraries/colordialog/lang/hu.js           |   11 +
 docroot/libraries/colordialog/lang/id.js           |   11 +
 docroot/libraries/colordialog/lang/is.js           |   11 +
 docroot/libraries/colordialog/lang/it.js           |   11 +
 docroot/libraries/colordialog/lang/ja.js           |   11 +
 docroot/libraries/colordialog/lang/ka.js           |   11 +
 docroot/libraries/colordialog/lang/km.js           |   11 +
 docroot/libraries/colordialog/lang/ko.js           |   11 +
 docroot/libraries/colordialog/lang/ku.js           |   11 +
 docroot/libraries/colordialog/lang/lt.js           |   11 +
 docroot/libraries/colordialog/lang/lv.js           |   11 +
 docroot/libraries/colordialog/lang/mk.js           |   11 +
 docroot/libraries/colordialog/lang/mn.js           |   11 +
 docroot/libraries/colordialog/lang/ms.js           |   11 +
 docroot/libraries/colordialog/lang/nb.js           |   11 +
 docroot/libraries/colordialog/lang/nl.js           |   11 +
 docroot/libraries/colordialog/lang/no.js           |   11 +
 docroot/libraries/colordialog/lang/oc.js           |   11 +
 docroot/libraries/colordialog/lang/pl.js           |   11 +
 docroot/libraries/colordialog/lang/pt-br.js        |   11 +
 docroot/libraries/colordialog/lang/pt.js           |   11 +
 docroot/libraries/colordialog/lang/ro.js           |   11 +
 docroot/libraries/colordialog/lang/ru.js           |   11 +
 docroot/libraries/colordialog/lang/si.js           |   11 +
 docroot/libraries/colordialog/lang/sk.js           |   11 +
 docroot/libraries/colordialog/lang/sl.js           |   11 +
 docroot/libraries/colordialog/lang/sq.js           |   11 +
 docroot/libraries/colordialog/lang/sr-latn.js      |   11 +
 docroot/libraries/colordialog/lang/sr.js           |   11 +
 docroot/libraries/colordialog/lang/sv.js           |   11 +
 docroot/libraries/colordialog/lang/th.js           |   11 +
 docroot/libraries/colordialog/lang/tr.js           |   11 +
 docroot/libraries/colordialog/lang/tt.js           |   11 +
 docroot/libraries/colordialog/lang/ug.js           |   11 +
 docroot/libraries/colordialog/lang/uk.js           |   11 +
 docroot/libraries/colordialog/lang/vi.js           |   11 +
 docroot/libraries/colordialog/lang/zh-cn.js        |   11 +
 docroot/libraries/colordialog/lang/zh.js           |   11 +
 docroot/libraries/colordialog/plugin.js            |   71 +
 docroot/libraries/glyphicons/css/config.json       |  402 ++++
 docroot/libraries/glyphicons/css/css/bootstrap.css |  819 +++++++
 .../css/fonts/glyphicons-halflings-regular.eot     |  Bin 0 -> 20127 bytes
 .../css/fonts/glyphicons-halflings-regular.svg     |  288 +++
 .../css/fonts/glyphicons-halflings-regular.ttf     |  772 +++++++
 .../css/fonts/glyphicons-halflings-regular.woff    |  Bin 0 -> 23424 bytes
 .../css/fonts/glyphicons-halflings-regular.woff2   |  Bin 0 -> 18028 bytes
 docroot/libraries/glyphicons/css/js/bootstrap.js   | 2366 ++++++++++++++++++++
 .../libraries/glyphicons/css/js/bootstrap.min.js   |   12 +
 docroot/libraries/glyphicons/css/style.css         |   13 +
 docroot/libraries/glyphicons/dialogs/glyphicons.js |  141 ++
 docroot/libraries/glyphicons/icons/glyphicons.png  |    6 +
 docroot/libraries/glyphicons/plugin.js             |   40 +
 85 files changed, 6052 insertions(+)
 create mode 100644 docroot/libraries/colordialog/dialogs/colordialog.css
 create mode 100644 docroot/libraries/colordialog/dialogs/colordialog.js
 create mode 100644 docroot/libraries/colordialog/lang/af.js
 create mode 100644 docroot/libraries/colordialog/lang/ar.js
 create mode 100644 docroot/libraries/colordialog/lang/az.js
 create mode 100644 docroot/libraries/colordialog/lang/bg.js
 create mode 100644 docroot/libraries/colordialog/lang/bn.js
 create mode 100644 docroot/libraries/colordialog/lang/bs.js
 create mode 100644 docroot/libraries/colordialog/lang/ca.js
 create mode 100644 docroot/libraries/colordialog/lang/cs.js
 create mode 100644 docroot/libraries/colordialog/lang/cy.js
 create mode 100644 docroot/libraries/colordialog/lang/da.js
 create mode 100644 docroot/libraries/colordialog/lang/de-ch.js
 create mode 100644 docroot/libraries/colordialog/lang/de.js
 create mode 100644 docroot/libraries/colordialog/lang/el.js
 create mode 100644 docroot/libraries/colordialog/lang/en-au.js
 create mode 100644 docroot/libraries/colordialog/lang/en-ca.js
 create mode 100644 docroot/libraries/colordialog/lang/en-gb.js
 create mode 100644 docroot/libraries/colordialog/lang/en.js
 create mode 100644 docroot/libraries/colordialog/lang/eo.js
 create mode 100644 docroot/libraries/colordialog/lang/es.js
 create mode 100644 docroot/libraries/colordialog/lang/et.js
 create mode 100644 docroot/libraries/colordialog/lang/eu.js
 create mode 100644 docroot/libraries/colordialog/lang/fa.js
 create mode 100644 docroot/libraries/colordialog/lang/fi.js
 create mode 100644 docroot/libraries/colordialog/lang/fo.js
 create mode 100644 docroot/libraries/colordialog/lang/fr-ca.js
 create mode 100644 docroot/libraries/colordialog/lang/fr.js
 create mode 100644 docroot/libraries/colordialog/lang/gl.js
 create mode 100644 docroot/libraries/colordialog/lang/gu.js
 create mode 100644 docroot/libraries/colordialog/lang/he.js
 create mode 100644 docroot/libraries/colordialog/lang/hi.js
 create mode 100644 docroot/libraries/colordialog/lang/hr.js
 create mode 100644 docroot/libraries/colordialog/lang/hu.js
 create mode 100644 docroot/libraries/colordialog/lang/id.js
 create mode 100644 docroot/libraries/colordialog/lang/is.js
 create mode 100644 docroot/libraries/colordialog/lang/it.js
 create mode 100644 docroot/libraries/colordialog/lang/ja.js
 create mode 100644 docroot/libraries/colordialog/lang/ka.js
 create mode 100644 docroot/libraries/colordialog/lang/km.js
 create mode 100644 docroot/libraries/colordialog/lang/ko.js
 create mode 100644 docroot/libraries/colordialog/lang/ku.js
 create mode 100644 docroot/libraries/colordialog/lang/lt.js
 create mode 100644 docroot/libraries/colordialog/lang/lv.js
 create mode 100644 docroot/libraries/colordialog/lang/mk.js
 create mode 100644 docroot/libraries/colordialog/lang/mn.js
 create mode 100644 docroot/libraries/colordialog/lang/ms.js
 create mode 100644 docroot/libraries/colordialog/lang/nb.js
 create mode 100644 docroot/libraries/colordialog/lang/nl.js
 create mode 100644 docroot/libraries/colordialog/lang/no.js
 create mode 100644 docroot/libraries/colordialog/lang/oc.js
 create mode 100644 docroot/libraries/colordialog/lang/pl.js
 create mode 100644 docroot/libraries/colordialog/lang/pt-br.js
 create mode 100644 docroot/libraries/colordialog/lang/pt.js
 create mode 100644 docroot/libraries/colordialog/lang/ro.js
 create mode 100644 docroot/libraries/colordialog/lang/ru.js
 create mode 100644 docroot/libraries/colordialog/lang/si.js
 create mode 100644 docroot/libraries/colordialog/lang/sk.js
 create mode 100644 docroot/libraries/colordialog/lang/sl.js
 create mode 100644 docroot/libraries/colordialog/lang/sq.js
 create mode 100644 docroot/libraries/colordialog/lang/sr-latn.js
 create mode 100644 docroot/libraries/colordialog/lang/sr.js
 create mode 100644 docroot/libraries/colordialog/lang/sv.js
 create mode 100644 docroot/libraries/colordialog/lang/th.js
 create mode 100644 docroot/libraries/colordialog/lang/tr.js
 create mode 100644 docroot/libraries/colordialog/lang/tt.js
 create mode 100644 docroot/libraries/colordialog/lang/ug.js
 create mode 100644 docroot/libraries/colordialog/lang/uk.js
 create mode 100644 docroot/libraries/colordialog/lang/vi.js
 create mode 100644 docroot/libraries/colordialog/lang/zh-cn.js
 create mode 100644 docroot/libraries/colordialog/lang/zh.js
 create mode 100644 docroot/libraries/colordialog/plugin.js
 create mode 100644 docroot/libraries/glyphicons/css/config.json
 create mode 100644 docroot/libraries/glyphicons/css/css/bootstrap.css
 create mode 100644 docroot/libraries/glyphicons/css/fonts/glyphicons-halflings-regular.eot
 create mode 100644 docroot/libraries/glyphicons/css/fonts/glyphicons-halflings-regular.svg
 create mode 100644 docroot/libraries/glyphicons/css/fonts/glyphicons-halflings-regular.ttf
 create mode 100644 docroot/libraries/glyphicons/css/fonts/glyphicons-halflings-regular.woff
 create mode 100644 docroot/libraries/glyphicons/css/fonts/glyphicons-halflings-regular.woff2
 create mode 100644 docroot/libraries/glyphicons/css/js/bootstrap.js
 create mode 100644 docroot/libraries/glyphicons/css/js/bootstrap.min.js
 create mode 100644 docroot/libraries/glyphicons/css/style.css
 create mode 100644 docroot/libraries/glyphicons/dialogs/glyphicons.js
 create mode 100644 docroot/libraries/glyphicons/icons/glyphicons.png
 create mode 100644 docroot/libraries/glyphicons/plugin.js
[workspace] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/cibox/jobs/deploy_code.yml -i /tmp/inventory6977123436823729151.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace -e build_number=1560 -e server_docroot_folder=/var/www -e deploy_type=local -e profile_source=/var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode -e source=/var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/docroot -e profile_destination=profiles/contrib/openy -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Generate commentinfo.md file] ************ 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Make sure folder is not exist] *********** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Sync files into docroot] ***************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Sync OpenY profile] ********************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Folder permissions] ********************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=6    changed=4    unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/cibox/jobs/build.yml -i /tmp/inventory8405038644940061359.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace -e build_number=1560 -e server_docroot_folder=/var/www -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Folder permissions] **************************************************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=2    changed=1    unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/www/build1560/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_password=root -e mysql_db=drupal1560 -e drupal_folder=/var/www/build1560 -e site_url=http://openy-dev.ffwua.com/build1560 -e pp_environment=default -e run_reinstall=true -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Include specific variables per environment] **************************** 
ok: [localhost]

TASK: [Getting timestamp for backup] ****************************************** 
skipping: [localhost]

TASK: [Getting name for backup] *********************************************** 
skipping: [localhost]

TASK: [Creating backup folder for environment] ******************************** 
skipping: [localhost]

TASK: [Making regular backup from {{ local_backup_environment }} environment] *** 
skipping: [localhost]

TASK: [Make new backup as latest] ********************************************* 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
changed: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
changed: [localhost]

TASK: [Set writable private dir] ********************************************** 
changed: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
skipping: [localhost]

TASK: [Remove settings.php] *************************************************** 
skipping: [localhost]

TASK: [Creating settings.php] ************************************************* 
skipping: [localhost]

TASK: [Adding database variable to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add general pre-settings to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add environment specific pre-settings] ********************************* 
skipping: [localhost] => (item={'status': True, 'name': '$settings["hash_salt"]', 'value': 'g1W4KZgWCeWh7Af9_gSb9D9HOiNaBdWPvQD-8x0yWpcgNpTBppaHR5VNGQBGvh2k8P8uWa1V8w'})
skipping: [localhost] => (item={'status': False, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
skipping: [localhost] => (item={'status': True, 'name': '$config_directories["sync"]', 'value': 'sites/default/config/sync'})
skipping: [localhost] => (item={'status': True, 'name': '$settings["install_profile"]', 'value': u'openy'})
skipping: [localhost] => (item={'status': True, 'name': '$settings["file_public_path"]', 'value': u'sites/default/files'})
skipping: [localhost] => (item={'status': True, 'name': '$config["system.file"]["path.temporary"]', 'value': '/tmp'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
skipping: [localhost]

TASK: [Creating drupal database] ********************************************** 
skipping: [localhost]

TASK: [Downloading latest backup to local environment] ************************ 
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
skipping: [localhost]

TASK: [Importing from local backup] ******************************************* 
skipping: [localhost]

TASK: [Remove local backup from build folder] ********************************* 
skipping: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
skipping: [localhost]

TASK: [Clear drush cache] ***************************************************** 
skipping: [localhost]

TASK: [Rebuilding drupal registry] ******************************************** 
skipping: [localhost]

TASK: [Clear Drupal cache] **************************************************** 
skipping: [localhost]

TASK: [Updating database] ***************************************************** 
skipping: [localhost]

TASK: [Apply pending entity schema updates] *********************************** 
skipping: [localhost]

TASK: [Download stage file proxy] ********************************************* 
skipping: [localhost]

TASK: [Enable stage file proxy] *********************************************** 
skipping: [localhost]

TASK: [Set configuration for stage file proxy] ******************************** 
skipping: [localhost]

TASK: [Set folder for stage file proxy] *************************************** 
skipping: [localhost]

TASK: [Updates listing] ******************************************************* 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
ok: [localhost]

TASK: [Creating drupal database] ********************************************** 
changed: [localhost]

TASK: [Remove settings.php] *************************************************** 
ok: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
changed: [localhost]

TASK: [Installing drupal] ***************************************************** 
changed: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
ok: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
ok: [localhost]

TASK: [Set writable private dir] ********************************************** 
ok: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
ok: [localhost]

TASK: [Copy settings to tmp folder (WINDOWS)] ********************************* 
skipping: [localhost]

TASK: [Add general post-settings to settings.php] ***************************** 
changed: [localhost] => (item={'status': True, 'name': '$settings["pp_environment"]', 'value': u'default'})

TASK: [Add environment specific post-settings] ******************************** 
changed: [localhost] => (item={'status': True, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
skipping: [localhost] => (item={'status': False, 'name': '$settings["cache"]["bins"]["render"]', 'value': 'cache.backend.memory'})
skipping: [localhost] => (item={'status': False, 'name': '$config["stage_file_proxy.settings"]["origin"]', 'value': u'http://openy-dev.ffwua.com/demo_upgrade_path'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Enable general modules] ************************************************ 
skipping: [localhost]

TASK: [Enable environment specific modules] *********************************** 
skipping: [localhost] => (item={'status': False, 'name': 'devel shield'})

TASK: [Run general drush commands] ******************************************** 
skipping: [localhost]

TASK: [Run environment specific drush commands] ******************************* 
changed: [localhost] => (item={'status': True, 'name': 'pm-uninstall', 'arguments': 'update -y'})
changed: [localhost] => (item={'status': True, 'name': 'user-password', 'arguments': u'admin --password=openy'})
changed: [localhost] => (item={'status': True, 'name': 'cr', 'arguments': ''})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': u'shield.settings user admin -y'})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': u'shield.settings pass openy -y'})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': 'system.site uuid USER_YOUR_UUID_HERE -y'})
skipping: [localhost] => (item={'status': False, 'name': 'config-set', 'arguments': u'stage_file_proxy.settings origin http://openy-dev.ffwua.com/demo_upgrade_path -y'})
changed: [localhost] => (item={'status': True, 'name': 'config-set', 'arguments': "geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y"})

TASK: [Debug] ***************************************************************** 
ok: [localhost] => (item={'cmd': 'APP_ENV=dev drush -y pm-uninstall update -y', 'end': '2017-06-14 19:41:39.098858', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.\nupdate was successfully uninstalled.                                 \x1b[1;32;40m\x1b[1m[ok]\x1b[0m', 'stdout': 'The following extensions will be uninstalled: update\nDo you really want to continue? (y/n): y', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'pm-uninstall', 'arguments': 'update -y'}, 'warnings': [], 'delta': '0:00:05.104877', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u'APP_ENV=dev drush -y pm-uninstall update -y'}, 'stdout_lines': ['The following extensions will be uninstalled: update', 'Do you really want to continue? (y/n): y'], 'start': '2017-06-14 19:41:33.993981'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y pm-uninstall update -y",
        "delta": "0:00:05.104877",
        "end": "2017-06-14 19:41:39.098858",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y pm-uninstall update -y",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "update -y",
            "name": "pm-uninstall",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-14 19:41:33.993981",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.\nupdate was successfully uninstalled.                                 \u001b[1;32;40m\u001b[1m[ok]\u001b[0m",
        "stdout": "The following extensions will be uninstalled: update\nDo you really want to continue? (y/n): y",
        "stdout_lines": [
            "The following extensions will be uninstalled: update",
            "Do you really want to continue? (y/n): y"
        ],
        "warnings": []
    },
    "var": {
        "item.stdout": "The following extensions will be uninstalled: update\nDo you really want to continue? (y/n): y"
    }
}
ok: [localhost] => (item={'cmd': 'APP_ENV=dev drush -y user-password admin --password=openy', 'end': '2017-06-14 19:41:40.588595', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.\nChanged password for admin                                           \x1b[1;32;40m\x1b[1m[success]\x1b[0m', 'stdout': u'', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'user-password', 'arguments': u'admin --password=openy'}, 'warnings': [], 'delta': '0:00:01.401771', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u'APP_ENV=dev drush -y user-password admin --password=openy'}, 'stdout_lines': [], 'start': '2017-06-14 19:41:39.186824'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y user-password admin --password=openy",
        "delta": "0:00:01.401771",
        "end": "2017-06-14 19:41:40.588595",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y user-password admin --password=openy",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "admin --password=openy",
            "name": "user-password",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-14 19:41:39.186824",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.\nChanged password for admin                                           \u001b[1;32;40m\u001b[1m[success]\u001b[0m",
        "stdout": "",
        "stdout_lines": [],
        "warnings": []
    },
    "var": {
        "item.stdout": ""
    }
}
ok: [localhost] => (item={'cmd': 'APP_ENV=dev drush -y cr', 'end': '2017-06-14 19:41:45.366105', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.\nCache rebuild complete.                                              \x1b[1;32;40m\x1b[1m[ok]\x1b[0m\nDirectory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.', 'stdout': u'', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'cr', 'arguments': ''}, 'warnings': [], 'delta': '0:00:04.702406', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u'APP_ENV=dev drush -y cr'}, 'stdout_lines': [], 'start': '2017-06-14 19:41:40.663699'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y cr",
        "delta": "0:00:04.702406",
        "end": "2017-06-14 19:41:45.366105",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y cr",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "",
            "name": "cr",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-14 19:41:40.663699",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.\nCache rebuild complete.                                              \u001b[1;32;40m\u001b[1m[ok]\u001b[0m\nDirectory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.",
        "stdout": "",
        "stdout_lines": [],
        "warnings": []
    },
    "var": {
        "item.stdout": ""
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={u'skipped': True, u'changed': False}) => {
    "item": {
        "changed": false,
        "skipped": true
    },
    "var": {
        "item.stdout": "{{item.stdout}}"
    }
}
ok: [localhost] => (item={'cmd': "APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y", 'end': '2017-06-14 19:41:46.519120', 'stderr': 'Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \x1b[31;40m\x1b[1m[error]\x1b[0m\nwritable. Please check directory permissions.', 'stdout': 'Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y', 'changed': True, 'rc': 0, 'item': {'status': True, 'name': 'config-set', 'arguments': "geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y"}, 'warnings': [], 'delta': '0:00:00.959892', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u"APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y"}, 'stdout_lines': ['Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y'], 'start': '2017-06-14 19:41:45.559228'}) => {
    "item": {
        "changed": true,
        "cmd": "APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y",
        "delta": "0:00:00.959892",
        "end": "2017-06-14 19:41:46.519120",
        "invocation": {
            "module_args": "APP_ENV=dev drush -y config-set geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y",
            "module_complex_args": {},
            "module_name": "shell"
        },
        "item": {
            "arguments": "geolocation.settings google_map_api_key 'AIzaSyDIFPdDmDsBOFFZcmavCaAqHa3VNKkXLJc' -y",
            "name": "config-set",
            "status": true
        },
        "rc": 0,
        "start": "2017-06-14 19:41:45.559228",
        "stderr": "Directory /var/lib/jenkins/.drush/cache/default exists, but is not   \u001b[31;40m\u001b[1m[error]\u001b[0m\nwritable. Please check directory permissions.",
        "stdout": "Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y",
        "stdout_lines": [
            "Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y"
        ],
        "warnings": []
    },
    "var": {
        "item.stdout": "Do you want to update google_map_api_key key in geolocation.settings config? (y/n): y"
    }
}

PLAY RECAP ******************************************************************** 
localhost                  : ok=28   changed=11   unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/www/build1560/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_password=root -e mysql_db=drupal_upgrade1560 -e drupal_folder=/var/www/build1560 -e site_url=http://upgrade.openy-dev.ffwua.com/build1560 -e pp_environment=default -e run_reinstall=true -e workflow_type=sql -e settings_site_folder=upgrade -e trigger_drush_make=false -e stage_file_proxy=true -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Include specific variables per environment] **************************** 
ok: [localhost]

TASK: [Getting timestamp for backup] ****************************************** 
skipping: [localhost]

TASK: [Getting name for backup] *********************************************** 
skipping: [localhost]

TASK: [Creating backup folder for environment] ******************************** 
skipping: [localhost]

TASK: [Making regular backup from {{ local_backup_environment }} environment] *** 
skipping: [localhost]

TASK: [Make new backup as latest] ********************************************* 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
changed: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
changed: [localhost]

TASK: [Set writable private dir] ********************************************** 
changed: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
ok: [localhost]

TASK: [Remove settings.php] *************************************************** 
ok: [localhost]

TASK: [Creating settings.php] ************************************************* 
changed: [localhost]

TASK: [Adding database variable to settings.php] ****************************** 
changed: [localhost]

TASK: [Add general pre-settings to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add environment specific pre-settings] ********************************* 
changed: [localhost] => (item={'status': True, 'name': '$settings["hash_salt"]', 'value': 'g1W4KZgWCeWh7Af9_gSb9D9HOiNaBdWPvQD-8x0yWpcgNpTBppaHR5VNGQBGvh2k8P8uWa1V8w'})
skipping: [localhost] => (item={'status': False, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
changed: [localhost] => (item={'status': True, 'name': '$config_directories["sync"]', 'value': 'sites/default/config/sync'})
changed: [localhost] => (item={'status': True, 'name': '$settings["install_profile"]', 'value': u'openy'})
changed: [localhost] => (item={'status': True, 'name': '$settings["file_public_path"]', 'value': u'sites/upgrade/files'})
changed: [localhost] => (item={'status': True, 'name': '$config["system.file"]["path.temporary"]', 'value': '/tmp'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
ok: [localhost]

TASK: [Creating drupal database] ********************************************** 
changed: [localhost]

TASK: [Downloading latest backup to local environment] ************************ 
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
fatal: [localhost] => input file not found at /var/www/backup/latest_release.sql.gz or /var/www/backup/latest_release.sql.gz

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=16   changed=8    unreachable=1    failed=0   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

Sanchiz commented:
retest this please

gundevel commented:
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1575 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1575 |
| Installation process | http://install.openy-dev.ffwua.com/build1575/install.php |

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1577/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #434 of commit d564906e7f0d2de1490fac5079f6e0719854a90e automatically merged.
GitHub pull request #434 of commit d564906e7f0d2de1490fac5079f6e0719854a90e automatically merged.
GitHub pull request #434 of commit d564906e7f0d2de1490fac5079f6e0719854a90e automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
Resetting working tree
 > git reset --hard # timeout=10
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git3896243130475336588.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git rev-parse origin/pr/434/merge^{commit} # timeout=10
 > git branch -a --contains 713a36b4ab8f614a8e6b9a4d154b91dc108156a8 # timeout=10
 > git rev-parse remotes/origin/pr/434/merge^{commit} # timeout=10
Checking out Revision 713a36b4ab8f614a8e6b9a4d154b91dc108156a8 (origin/pr/434/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 713a36b4ab8f614a8e6b9a4d154b91dc108156a8
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson2024594958596396399.sh
+ rm -Rf /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm
+ cp -R ../vm /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm
+ cd /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm
+ git pull
fatal: The remote end hung up unexpectedly
Terminated
Build was aborted
Aborted by anonymous
```

Sanchiz commented:
retest this please

bjwilkins commented:
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1580 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1580 |
| Installation process | http://install.openy-dev.ffwua.com/build1580/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1580/

bjwilkins commented:
Behat test results file URL: http://ci.openymca.org/build1580/profiles/contrib/openy/build/reports/behat

AlexNetman commented:
Safari issues:
![image](https://user-images.githubusercontent.com/4558228/27231288-d46b1550-52ba-11e7-9a35-3bc294cb06c2.png)

* Arrows are cut
* Tab navigate through those arrows and no chance to jump out of the loop (let's resolve at least part of the issue - seems safari is watching on Tabindex attributes, that's why only some elements are available for navigation.

AlexNetman commented:
retest this please

bjwilkins commented:
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1616 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1616 |
| Installation process | http://install.openy-dev.ffwua.com/build1616/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1616/

bjwilkins commented:
Behat test results file URL: http://ci.openymca.org/build1616/profiles/contrib/openy/build/reports/behat

david-hernandez committed:
Adding href to the a tag after changing from button. This seem the best for getting it clickable. Not all are responing to role button

david-hernandez commented:
The last commit should help with the tabbing through the arrow links. I get good results in chrome. I am not able to use Safari at all to test this. I don't know if it it my Safari or what, but it does not tab properly on any website at all. 

On Chrome I can successfully navigate through the arrows and transition the slides.

bjwilkins commented:
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1643 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1643 |
| Installation process | http://install.openy-dev.ffwua.com/build1643/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1643/

bjwilkins commented:
Behat test results file URL: http://ci.openymca.org/build1643/profiles/contrib/openy/build/reports/behat
